scripts/component_generation/go.mod

```diff
@@ -0,0 +1,136 @@
+module github.com/sudo-NithishKarthik/meshmodel_ahub_gen
```

Comment:
Will need to land this module elsewhere.

Reply:
moved to `layer5io/meshmodel_ahub_gen`